docs/releases/4.2.txt

```diff
@@ -26,6 +26,15 @@ and only officially support the latest release of each series.
 What's new in Django 4.2
 ========================
 
+Asynchronous Model interface
+----------------------------
+
+``Model`` now provides an asynchronous interface for some model methods that
+user database. These are named as -per the existing synchronous operations but
+with an ``a`` prefix, for example ``asave()``, ``adelete()``, and so on.
+
+see :ref:`async-model-methods` for details.
```

Comment:
This is a much smaller change than the async QuerySet API, so I think it should be a list item under the Models heading (line 215). Separate sections are reserved for headline features.